src/main/downloadsManager.ts

```diff
@@ -174,14 +174,19 @@ export class DownloadsManager extends JsonFileManager<DownloadedItems> {
         }
 
         for (const file of Object.values(this.downloads)) {
-            if (file.bookmark) {
-                this.bookmarks.set(this.getDownloadedFileId(file), {originalPath: file.location, bookmark: file.bookmark});
+            try {
```

Comment:
Should we minimize the try/catch to only be around whatever we expect to throw the error? 0/5

Reply:
I want to make sure we also try/catch the bookmark call, it's another system call that has potential to fail. I think we're okay to do it all.